core/state_transition.go

```diff
@@ -605,12 +605,17 @@ func (st *stateTransition) innerExecute() (*ExecutionResult, error) {
 	// is always 0 for deposit tx. So calling refundGas will ensure the gasUsed accounting is correct without actually
 	// changing the sender's balance
 	var gasRefund uint64
+	var worstOperatorCost *big.Int = nil
```

Comment:
small style nit, not need to do equals nil, it will just be nil

Reply:
Done